### ISSUE TITLE
What about including video codec selection?

### ISSUE THREAD

**Issue by Gabe-LS** [rse/obs-ninja-trampoline#1]
Hello @rse, it's me again. I just discovered this other project of yours. Can I know why you decided to completely exclude VP8/9 codecs? In my experience, they offer much better colors. And VP9 requires less bandwidth to gives you the same result (of course, it also requires more CPU power).

**Comment by rse** [comment]
Indeed, VP9 is a real CPU eater and usually does not benefit from GPU acceleration. That's why I originally switched to just H264. But if wished, I can add an option to allow one to select the codec again.
